test/remote/gateways/remote_mit_test.rb

```diff
@@ -0,0 +1,127 @@
+require 'test_helper'
+
+class RemoteMitTest < Test::Unit::TestCase
+  def setup
+    @gateway = MitGateway.new(fixtures(:mit))
+
+    @amount = 1115
+    @amount_fail = 11165
+
+    @credit_card = ActiveMerchant::Billing::CreditCard.new(
+      number: '4242424242424242',
+      verification_value: '261',
+      month: '09',
+      year: '2025',
+      first_name: 'Pedro',
+      last_name: 'Flores Valdes'
+    )
+
+    @declined_card = ActiveMerchant::Billing::CreditCard.new(
+      number: '4242424242424242',
+      verification_value: '318',
+      month: '09',
+      year: '2025',
+      first_name: 'Pedro',
+      last_name: 'Flores Valdes'
+    )
+
+    @options_success = {
+      order_id: '721',
+      transaccion_id: '721', # unique id for every transaction, needs to be generated for every test
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+
+    @options = {
+      order_id: '721',
+      transaccion_id: '721', # unique id for every transaction, needs to be generated for every test
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+  end
+
+  def test_successful_purchase
+    puts('======================================================================================')
```

Comment:
Similar to above, we would like to not have `puts` statements in any of our test code.